index.js

```diff
@@ -81,6 +81,9 @@ module.exports = {
       destDir: appImagesDirectory
     });
 
+    // Leaflet-Omnivore.
+    app.import(app.bowerDirectory + '/leaflet-omnivore/leaflet-omnivore.min.js');
```

Comment:
You should include src files, cause we should can debug it. Ember will minify result if were built with production.